resources/AnVIL_Template_CollectionStatus_Table.html

```diff
@@ -0,0 +1,230 @@
+<table>
+  <tr>
+    <th>Repo name</th>
+    <th>Render/Package Status</th>
+    <th>Number of Open Issues</th>
+    <th>Number of branches</th>
+    <th>Number of Open PRs</th>
+    <th>Website up?</th>
+  </tr>
+  <!--
```

Comment:
I like this comment showing what the format is! Maybe just a few words mentioning that?